src/main/java/com/yanolja_final/domain/user/controller/EmailController.java

```diff
@@ -0,0 +1,46 @@
+package com.yanolja_final.domain.user.controller;
+
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import lombok.RequiredArgsConstructor;
+import com.yanolja_final.domain.user.dto.request.EmailRequest;
+import com.yanolja_final.domain.user.service.EmailService;
+import com.yanolja_final.global.util.ResponseDTO;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/email")
```

Comment:
```suggestion
@RequestMapping("/v1/email")
```

이렇게 바꾸는 거 어떠신가요?
`/api/v1`로 시작하도록 하는 사례가 많이 보여서
저희는 서브도메인에서 api.winnerone.site 이렇게 api가 이미 들어가서 /v1만 넣는 식으로 제 로그인 코드에서 적용해놨거든요

Reply:
엔드포인트 변경하여 커밋하였습니다!